### ISSUE TITLE
IC2 OLED DIsplay

### ISSUE THREAD

**Issue by g6hfs** [tomvdb/open_tuner#18]
Tom,

Will there be any support for IC2 oled displays.

Brian

![image](https://user-images.githubusercontent.com/108218059/227185309-9c733b07-dbcb-4668-9247-d76bf68de3f8.jpeg)

**Comment by tomvdb** [comment]
The concept is fairly simple and can be done, but I don't currently have access to the hardware for this. As soon as I get the hardware, i'll add it in for sure. Consider it on the list.

**Comment by g6hfs** [comment]
Tom,Thankyou for that.Brian
-------- Original message --------From: Tom Van den Bon ***@***.***> Date: 23/03/2023  18:39  (GMT+00:00) To: tomvdb/open_tuner ***@***.***> Cc: g6hfs ***@***.***>, Author ***@***.***> Subject: Re: [tomvdb/open_tuner] IC2 OLED DIsplay (Issue #18) 
The concept is fairly simple and can be done, but I don't currently have access to the hardware for this. As soon as I get the hardware, i'll add it in for sure. Consider it on the list.

**Comment by 9H1LO** [comment]
yes having OLED working would be cool, let us know if you need an oled sending Tom
